examples/integration/.bazelrc

```diff
@@ -11,6 +11,7 @@ build:rules_xcodeproj --@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:ext
 build:rules_xcodeproj_integration --define=foo=bar
 
 build --test_env=INTEGRATION_TEST_ENV_VAR=TRUE
+build --action_env=INTEGRATION_ACTION_ENV_VAR=TRUE
```

Comment:
I don't believe this is used the same way as `test_env`, and is only exposed to build actions. Let's remove it.